src/helpers/format.ts

```diff
@@ -42,9 +52,45 @@ export function formatYesNoQueryString(str: string) {
   const formatted = title.substring(0, title.length - 2);
 
   // Return only the first 55 characters
-  if (formatted.length > 55) {
-    return `${formatted.substring(0, 56)}...`;
+  if (formatted.length > maxLength) {
+    return `${formatted.substring(0, maxLength + 1)}...`;
   } else {
     return formatted;
   }
 }
+
+type FormatRequestTitleParams = {
+  identifier?: string;
+  ancillaryData?: string;
+  maxTitleLength?: number;
+  defaultTitle?: string;
+};
+
+export function formatRequestTitle(params: FormatRequestTitleParams) {
+  const {
+    identifier,
+    ancillaryData,
+    maxTitleLength = 55,
+    defaultTitle = "Optimistic Oracle Request",
+  } = params;
```

Comment:
I wonder if we should just pull these constants out of the function themselves, wdyt? Call out these defaults are configuable too (even if it's just above this func).

Reply:
maxTitleLength = 55. Set 55 to a constant.

Reply:
i see ok